submissions/progoogler.md

```diff
@@ -0,0 +1,10 @@
+* submitter: @progoogler
+* statement: I hereby agree that the facts I have submitted are my own, and that I grant a license for Pryaxis Limited, or designated entities by Pryaxis Limited, the ability to use this data in a way to promote future Pryaxis services.
+
+----
+
```

Comment:
You really need a ```Machine Learning:``` here or else each fact below doesn't stand in isolation (they don't explicitly reference ```@Machine Learning``` or ```@ML``` in 100% of the cases. Cheers!